eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -558,7 +558,8 @@
   <Match>
     <Or>
       <Class name="~.*JavaDoc(CodeSnippets|CodeSamples|Samples)"/>
-      <Class name="com.azure.storage.blob.batch.ReadmeCodeSamples"/>
+      <Class name="~.*ReadmeSamples"/>
+      <Class name="com.azure.storage.blob.batch.ReadmeSamples"/>
```

Comment:
You shouldn't need to explicitly call out the batch file should you?

Reply:
Removed it.